### PR TITLE
create main table

### PR DESCRIPTION
**Pull request by KirillGolenko**

### CODE REVIEW

File: src/Components/Complaints/Complaints.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    margin: 10px 0;
+  }
+
+  .main-add {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    margin: 0 2rem;
+    width: 25ch;
+  }
```
Comment (usatovaDasha) [inline src/Components/Complaints/Complaints.scss:21]: add empty line

File: src/Components/Main/Main.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import Container from '@material-ui/core/Container';
```
Comment (usatovaDasha) [inline src/Components/Main/Main.js:3]: core in one import

File: src/Components/Main/Main.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import Zoom from '@material-ui/core/Zoom';
+import Alert from '@material-ui/lab/Alert';
+
```
Comment (usatovaDasha) [inline src/Components/Main/Main.js:6]: remove empty line

File: src/Components/Main/Main.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+import Header from '../Header/Header';
+import Table from '../Table/Table';
+import Complaints from '../Complaints/Complaints';
+import './Main.scss';
```
Comment (usatovaDasha) [inline src/Components/Main/Main.js:10]: add empty line

File: src/Components/Main/Main.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+	};
+
+	useEffect(() => {
+		fetchData();
+	}, []);
+
+	return (
+		<>
+			<Header title='Приемы' />
+
```
Comment (usatovaDasha) [inline src/Components/Main/Main.js:41]: remove empty line

File: src/Components/Register/Register.js [hunk 6]
```diff
@@ -11,20 +12,43 @@ import Typography from '@material-ui/core/Typography';
 import OutlinedInput from '@material-ui/core/OutlinedInput';
 import Button from '@material-ui/core/Button';
```
Comment (usatovaDasha) [inline src/Components/Register/Register.js:13]: core

File: src/Components/Register/Register.js [hunk 7]
```diff
@@ -36,72 +60,154 @@ const Auth = props => {
...
+	};
 
-				<FormControl>
-					<label>Password:</label>
+	const validText = async () => {
+		const currectLogin = values.login.match('[a-zA-Z0-9]{6,}$');
+		const currectPasword = values.password.match(
+			'(?=.*[A-Za-z])(?=.*[0-9]){6,}'
+		);
+		const currectRepeatPassword = values.repeatPassword === values.password;
```
Comment (usatovaDasha) [inline src/Components/Register/Register.js:76]: correctRepeatPassword

File: src/Components/Register/Register.js [hunk 8]
```diff
@@ -36,72 +60,154 @@ const Auth = props => {
...
+					alertMessage(error, 'error');
+				});
+		}
+	};
+	return (
+		<>
+			<Header title='Зарегистрироваться в системе' />
+			{error && (
+				<Zoom in={error}>
+					<Alert severity={modalValues.type} className='myAlert'>
```
Comment (usatovaDasha) [inline src/Components/Register/Register.js:111]: destructurization

File: src/Components/Register/Register.js [hunk 9]
```diff
@@ -36,72 +60,154 @@ const Auth = props => {
...
+								<OutlinedInput
+									placeholder='Password'
+									type={values.showPassword ? 'text' : 'password'}
+									value={values.password}
+									onChange={e => handleChange('password', e)}
+									endAdornment={
+										<InputAdornment position='end'>
+											<IconButton
+												aria-label='toggle password visibility'
+												onClick={handleClickShowPassword}
```
Comment (usatovaDasha) [inline src/Components/Register/Register.js:149]: correct call function

File: src/Components/Register/Register.js [hunk 10]
```diff
@@ -36,72 +60,154 @@ const Auth = props => {
...
+								<OutlinedInput
+									placeholder='Password'
+									type={values.showPassword ? 'text' : 'password'}
+									value={values.repeatPassword}
+									onChange={e => handleChange('repeatPassword', e)}
+									endAdornment={
+										<InputAdornment position='end'>
+											<IconButton
+												aria-label='toggle password visibility'
+												onClick={handleClickShowPassword}
```
Comment (usatovaDasha) [inline src/Components/Register/Register.js:176]: higher

File: src/Components/Register/Register.scss [hunk 11]
```diff
@@ -0,0 +1,111 @@
+.MuiAlert-root.myAlert {
+  position: absolute;
+  right: 0;
+  margin: 20px;
+  transition: 0.4;
+}
+
+.MuiContainer-root {
+  @media (max-width: 1440px) {
```
Comment (usatovaDasha) [inline src/Components/Register/Register.scss:9]: media higher

File: src/Components/Register/Register.scss [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+    .content__body,
+    .content__logo {
+      width: 40% !important;
```
Comment (usatovaDasha) [inline src/Components/Register/Register.scss:12]: remove important

File: src/Components/Register/Register.scss [hunk 13]
```diff
@@ -0,0 +1,111 @@
...
+    }
+  }
```
Comment (usatovaDasha) [inline src/Components/Register/Register.scss:14]: add empty line

File: src/Components/Register/Register.scss [hunk 14]
```diff
@@ -0,0 +1,111 @@
...
+  @media (max-width: 786px) {
+    &.content {
+      flex-direction: column !important;
+    }
```
Comment (usatovaDasha) [inline src/Components/Register/Register.scss:18]: add empty line

File: src/Components/Table/Table.js [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+	TableRow,
+	TableBody,
+} from '@material-ui/core';
+import Moment from 'react-moment';
+import edit_img from '../../Assets/image/edit.svg';
+import delete_img from '../../Assets/image/delete.svg';
+import './Table.scss';
+
+const TableComplaints = ({ allComplaints }) => {
+	// const [allComplaints, setComplaints] = useState([]);
```
Comment (usatovaDasha) [inline src/Components/Table/Table.js]: remove

File: src/Components/Table/Table.js [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+
+	return (
+		<div className='content__main-add'>
+			<TableContainer>
+				<Table>
+					<TableHead>
+						<TableRow>
+							<TableCell>Имя</TableCell>
```
Comment (usatovaDasha) [inline src/Components/Table/Table.js:21]: change on map

File: src/Components/Table/Table.js [hunk 17]
```diff
@@ -0,0 +1,61 @@
...
+							<TableCell>Врач</TableCell>
+							<TableCell>Дата</TableCell>
+							<TableCell>Жалобы</TableCell>
+							<TableCell></TableCell>
+						</TableRow>
+					</TableHead>
+					<TableBody>
+						{allComplaints.map(value => {
+							return (
+								<TableRow key={value._id}>
```
Comment (usatovaDasha) [inline src/Components/Table/Table.js:31]: destructurization

File: src/Components/Table/Table.js [hunk 18]
```diff
@@ -0,0 +1,61 @@
...
+									<TableCell>{value.name}</TableCell>
```
Comment (usatovaDasha) [inline src/Components/Table/Table.js:32]: and change on map

File: src/Components/Complaints/Complaints.js [hunk 19]
```diff
@@ -1,19 +1,14 @@
...
 import axios from 'axios';
-
-import OutlinedInput from '@material-ui/core/OutlinedInput';
-import Button from '@material-ui/core/Button';
-
-// import DateFnsUtils from '@date-io/date-fns';
-// import { MuiPickersUtilsProvider } from '@material-ui/pickers';
 import { DatePicker } from '@progress/kendo-react-dateinputs';
-import './Complaints.scss';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
```
Comment (usatovaDasha) [inline src/Components/Complaints/Complaints.js:4]: core

File: src/Components/Header/Header.js [hunk 20]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import AppBar from '@material-ui/core/AppBar';
-import Toolbar from '@material-ui/core/Toolbar';
 import Typography from '@material-ui/core/Typography';
 import Container from '@material-ui/core/Container';
+import Toolbar from '@material-ui/core/Toolbar';
```
Comment (usatovaDasha) [inline src/Components/Header/Header.js:4]: core

File: src/Components/Register/Register.js [hunk 21]
```diff
@@ -1,20 +1,20 @@
 import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import axios from 'axios';
-import IconButton from '@material-ui/core/IconButton';
 import InputAdornment from '@material-ui/core/InputAdornment';
-import FormControl from '@material-ui/core/FormControl';
-import Visibility from '@material-ui/icons/Visibility';
 import VisibilityOff from '@material-ui/icons/VisibilityOff';
-import Card from '@material-ui/core/Card';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
```
Comment (usatovaDasha) [inline src/Components/Register/Register.js:6]: core

File: src/Components/Table/Table.js [hunk 22]
```diff
@@ -1,20 +1,17 @@
...
-	Table,
-	TableHead,
-	TableCell,
-	TableRow,
-	TableBody,
-} from '@material-ui/core';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableCell from '@material-ui/core/TableCell';
+import TableBody from '@material-ui/core/TableBody';
```
Comment (usatovaDasha) [inline src/Components/Table/Table.js:5]: core

File: src/Components/Table/Table.js (hunk 18 above)
Comment (KirillGolenko) [reply src/Components/Table/Table.js:32]: there so map

File: src/Components/Complaints/Complaints.js [hunk 23]
```diff
@@ -1,10 +1,8 @@
 import React, { useState } from 'react';
 import axios from 'axios';
 import { DatePicker } from '@progress/kendo-react-dateinputs';
-import OutlinedInput from '@material-ui/core/OutlinedInput';
-import TextField from '@material-ui/core/TextField';
-import MenuItem from '@material-ui/core/MenuItem';
-import Button from '@material-ui/core/Button';
+import { OutlinedInput, TextField, MenuItem, Button } from '@material-ui/core';
```
Comment (usatovaDasha) [inline src/Components/Complaints/Complaints.js:4]: move on single lines

File: src/Components/Complaints/Complaints.scss [hunk 24]
```diff
@@ -1,45 +1,45 @@
 .content__main-add {
-  display: flex;
-  flex-direction: row;
-  background-color: #c5e9ff;
-  margin: 20px 0;
-  width: 100%;
-  justify-content: center;
-  padding: 20px 0;
-  box-shadow: 0px 10px 4px 2px #00000075;
+	display: flex;
```
Comment (usatovaDasha) [inline src/Components/Complaints/Complaints.scss:2]: 4 spaces?

File: src/Components/Header/Header.js [hunk 25]
```diff
@@ -1,9 +1,5 @@
 import React from 'react';
-import Typography from '@material-ui/core/Typography';
-import Container from '@material-ui/core/Container';
-import Toolbar from '@material-ui/core/Toolbar';
-import AppBar from '@material-ui/core/AppBar';
-import Box from '@material-ui/core/Box';
+import { Typography, Container, Toolbar, AppBar, Box } from '@material-ui/core';
```
Comment (usatovaDasha) [inline src/Components/Header/Header.js:2]: move on single lines

**Pull request merged**